.github/workflows/nodejs.yml

```diff
@@ -12,7 +12,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [10.21.0, 12.18.0, 14.x, 15.x]
+        node-version: [12.18.0, 14.x, 16.x]
```

Comment:
@gengjiawen is there a reason that it has 12.18.0 versus 12.x like the other branches?

Reply:
setup-nodejs not getting latest 12.x, so this write to 12.18 directly by @gabrielschulhof